### ISSUE TITLE
Language and Editorial Comments

### ISSUE THREAD

**Issue by GinoCanessa** [IHE/ITI.DSUBm#63]
For sanity, I am collecting all of my editorial and linguistic comments into a single issue.  Section numbers are shown for each instance individually below.  I believe *most* of these are Low or Medium priority, as they are not substantive.

* Volume 1
  * 1:54.1 DSUBm Actors, Transactions, and Content Modules
    * Table 1.54.1-1: Actors should link to the sections describing them
    * 1:54.1.1.1-1:54.1.1.3: "that support the" should be "that supports the"
    * 1:54.1.2: Transaction Descriptions: given that 1.2.1-1.2.5 are just links, could these sections be collapsed down to a table?
  * 1:54.2: Actor Options
    * "The [actor] that declares to support this option…" is hard to parse.  Perhaps something like "[Actor]s that declare support for this option…" or "The [actor] that declares support for this option…"?
    * "Options that may be selected for each actor…" is a bit awkward to parse given that the guide is static text.  Perhaps something more like "Optional functionality that may be implemented for each actor…"?
    * It would be nice to include a table showing the differences between the related options.  E.g., columns for each 54.2.1 - 54.2.6, rows for the ITIs, and 'x's in the ones that have them; or columns for each 54.2.1 - 54.2.6 and a single row with the ITIs listed; etc.
    * 1:54.2.1: DocumentReference Subscription for Minimal Update option
      * "This option extends the creation of a DocumentReference resource including also the update of the “status” and the delete events of a DocumentReference Resource." should be something like "This options extends the triggers for the creation of a DocumentReference resource to also include updates of the document “status” and delete events of a DocumentReference Resource." (extend triggers, not the creation itself).
      * Citations of ITI docs should be links.
    * 1:54.2.2 DocumentReference Subscription for Full Events option
      * "This option extends the creation of a DocumentReference resource including all possible update events and the delete events of a DocumentReference Resource." should be something like "This option extends the triggers for the creation of a DocumentReference resource to include all possible update events and delete events of a DocumentReference Resource." (extend triggers, not the creation itself).
      * Citations of ITI docs should be links.
    * 1:54.2.3 Basic Folder Subscription option
      * "This option permits to subscribe for Folder type List Resource and describe the basic trigger events considered for a Folder Subscription." should be something like "This option permits [subscribers/specific actor] to subscribe for events about Folder type List Resources and describes the basic trigger events considered for a Folder Subscription."
      * "This option includes the creation of a Folder type List Resource and a limited update event on a Folder type List Resource." should be something like "This option includes triggers about the creation and limited update events on a Folder type List Resource."
      * Citations of ITI docs should be links.
    * 1:54.2.4-1:54.2.6 have the same sentence structure issues and ITI linking request.
  * 1:54.4 DSUBm Overview
    * 1:54.4.1 Concepts: Missing spacing between paragraphs.
  * 1:54.5 Security Considerations
    * "The reader should also consider the indication included in Safety and Security section." should be something like "The reader should also consider the information included in Safety and Security section of the Subscription Backport IG."
* Volume 2
  * 2:3.110.4 Interactions
    * There is an 'update' interaction specified in the scope that is not shown in Figure 2:3.110.4-1
  * 2:3.11x.4 Interactions
    * The diagrams in `2:3.112.4 Interactions` and `2:3.113.4 Interactions` contain links to all the messages.  For consistency, I would recommend either adding links to 110.4, 111.4, and 114.4 or removing them from 112.4 and 113.3.
  * 2:3.111.5.3 Expected Actions
    * MIME types (application/fhir+[json|xml]) are not formatted as `code` elements.
  * 2:3.111.6.2 Message Semantics
    * HTTP response codes in the second paragraph are not formatted as `code` elements.
    * "Bundle.entry.response.status" is not formatted as a code element.
  * 2:3.111.8 Security Considerations, 2:3.112.15 Security Considerations, 2:3.113.10.5 Security Considerations, and 2:3.114.8.1 Security Audit Considerations
    * Recommended to avoid contractions ("It's") for non-native English speakers.
  * 2:3.111.8.1 Security Audit Considerations, 2:3.112.15.1 Security Audit Considerations, and 2:3.113.10.5.1 Security Audit Considerations
    * The content is repeated for two actors, suggest collapsing into a single paragraph that references both actors.
  * 2:3.112.11.2 Message Semantics
    * "the Bundle.type element valued history" is different than all the other sections: "the Bundle.type element set to history"
* Changes to Other IHE Specifications
  * IHE Technical Frameworks Document Metadata Subscription (DSUB) Profile
    * Table is shown as markdown source.

**Comment by scontento** [comment]
Hi @GinoCanessa 
 thank you for reporting the problems and sharing your opinion

We have published the branch 
[edit-post-pc](https://github.com/IHE/ITI.DSUBm/tree/edit-post-pc)

with the major part of your comments.   
We think we are going to wait next week to review together this issue during the f2f. 
After the review we intend to finalize the edits and close this issue.

**Comment by MaryLJ** [comment]
> Hi @GinoCanessa thank you for reporting the problems and sharing your opinion
> 
> We have published the branch [edit-post-pc](https://github.com/IHE/ITI.DSUBm/tree/edit-post-pc)
> 
> with the major part of your comments. We think we are going to wait next week to review together this issue during the f2f. After the review we intend to finalize the edits and close this issue.

If the committee thinks that any of the suggestions/changes should involve a change to the template, please let me know. Thank you.

**Comment by gpavan19** [comment]
except for:

1:54.1.2: Transaction Descriptions: given that 1.2.1-1.2.5 are just links, could these sections be collapsed down to a table?

which involves the IHE template, the remaining comments will be integrated

**Comment by gpavan19** [comment]
resolved in the last editorial integration
